packages/suite/src/constants/suite/firmware.ts

```diff
@@ -1,13 +1,18 @@
 import { FirmwareHashCheckError, FirmwareRevisionCheckError } from '@trezor/connect';
 
-export const skippedRevisionCheckErrors = [
+// These errors will be treated softly, with only a warning displayed instead of modal
+export const softRevisionCheckErrors = [
     'cannot-perform-check-offline',
     'other-error',
 ] satisfies FirmwareRevisionCheckError[];
 
+// These errors are omitted entirely
 export const skippedHashCheckErrors = [
     'check-skipped',
     'check-unsupported',
     // this could be serious, but it's also caught by revision check, which handles edge-cases better, so it's skipped here
     'unknown-release',
 ] satisfies FirmwareHashCheckError[];
+
+// These errors will be treated softly, with only a warning displayed instead of modal
+export const softHashCheckErrors = [...skippedHashCheckErrors];
```

Comment:
In next PR I'll only add `other-error`. That was kinda the motivation to cleanup this mess :laughing: